onebusaway-gtfs-transformer/src/test/java/org/onebusaway/gtfs_transformer/deferred/DeferredValueSupportTest.java

```diff
@@ -72,18 +72,6 @@ public void testResolveConverter() {
         String.class);
     assertSame(ConvertUtils.lookup(String.class), converter);
   }
-  
-  @Test
-  public void testResolveConverter_FieldMappingConverter() {
```

Comment:
Any specific reason to remove this test? It seems like we're not running them anyway.

Reply:
It fails to run in java 17 at all. The tests do run in the existing CI workflow, just using Java 8. Even though we're not running them in the build workflow, I wanted to get the tests to pass in Java 17 anyway (I made this change a while ago when trying to get jitpack to work, and figured I'd add it here to get everything passing just in case).

I mentioned it here, and got confirmation that Java 17 isn't explicitly supported and won't be for the foreseeable future, but this would need to be fixed regardless. I can add it back when that happens.
https://github.com/OneBusAway/onebusaway-gtfs-modules/issues/208